src/raftstore/store/snap.rs

```diff
@@ -988,26 +471,18 @@ mod v2 {
                 return Ok(());
             }
             for cf_file in &mut self.cf_files {
-                if plain_file_used(cf_file.cf) {
```

Comment:
I totally remove v1 related code in this PR to kill clear. CF_LOCK will return to use plain file in another PR.